examples/nextjs-cldimage/styles/Home.module.css

```diff
@@ -21,5 +21,29 @@
 
 .imageTitle {
   font-size: 1.4em;
-  margin: .4em 0 0;
-}
\ No newline at end of file
+  margin: 0.4em 0 0;
+}
+
+.header {
```

Comment:
fwiw im not opposed to moving these projects over to tailwind but we could also do that in another phase

Reply:
Tailwind makes more sense for a "modern" example. Do you wish me to do the migration?

Reply:
ive you're up for it, let's do it! we can work through everything on this example to figure out how we want everything to look if that sounds good, then once we're good here, you can just apply it to all the other projects

Reply:
Also what do you say about putting the **AI Generative Fill** section in the end? It will look better if that section is at the bottom but it's a feature that needs to be highlighted as well.

Because it has images with more height, it leaves a lot of empty space. 

![image](https://github.com/cloudinary-community/cloudinary-examples/assets/51731966/4268f30b-7cf5-4084-8a15-741879b2be10)

Reply:
not opposed to it. was thinking having the Generative Fill example higher would highlight it more as an interesting feature

Reply:
I feel the same. Maybe we can put it at the top. Or keep it in the middle like before, what do you say?